<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Central Drive Estimation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="TorquePlot" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max Value Reached</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="maxVal_lcd"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="download_pb">
      <property name="text">
       <string>Download Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
